jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -876,7 +876,7 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
 
                 # Fit Gaussian1D to radial profile data.
                 if self.fit_radial_profile:
-                    fitter = LevMarLSQFitter()
+                    fitter = TRFLSQFitter()

```

Comment:
@larrybradley , is this replacement reasonable or another one is better? Also see https://github.com/astropy/photutils/issues/1895

Reply:
That should work fine.  I don't think you are using parameter bounds, so `LMLSQFitter` is also an option.

Reply:
Thanks! Well, not yet. No idea if that is going to be future request or not. 😬

Reply:
is there any consequence on performance between the two for this use-case?

Reply:
Nothing that we have to worry about. I am using what Astropy now suggests in their docs. Tom R made that change upstream and he has been also working on modeling performance there, so he wouldn't recommend anything that would make things worse.

Reply:
In my testing, `TRFLSQFitter` is about 4-5% slower than `LevMarLSQFitter`.  However, this difference will not be noticeable here.